rust/processor/src/db/common/models/account_transaction_models/account_transactions.rs

```diff
@@ -71,17 +73,21 @@ impl AccountTransaction {
             ),
             TxnData::Genesis(inner) => (&inner.events, vec![]),
             TxnData::BlockMetadata(inner) => (&inner.events, vec![]),
-            TxnData::Validator(inner) => (&inner.events, vec![]),
+            // No events in Movement protobuf Validator Tx.
```

Comment:
Could you expand on the comment here a bit?

Reply:
The initial indexer project was build with the rev `5c48aee129b5a141be2792ffa3d9bd0a1a61c9cb` of Aptos core that is after movement aptos-core initial fork. In this rev the protobuf schema has been changed, and they have added the events fields to the Tx. To build, I remove all events reference.